src/action/MoveAbovePlate.cpp

```diff
@@ -17,29 +17,19 @@ bool moveAbovePlate(
   double verticalTolerance = feedingDemo->mPlateTSRParameters.at("verticalTolerance");
   // NOTE: Although rotationTolerance was originally passed in as a param, it was never used.
   // double rotationTolerance = feedingDemo->mPlateTSRParameters.at("rotationTolerance");
-  const Eigen::Vector6d& velocityLimits = feedingDemo->mVelocityLimits;
-
-  // Hardcoded pose
-  Eigen::VectorXd homeConfig(6);
-  homeConfig << -2.11666, 3.34967, 2.04129, -2.30031, -2.34026, 2.9545;
-  bool success = ada->moveArmToConfiguration(
-      homeConfig, collisionFree, 2.0, velocityLimits);
-  if (!success)
+  auto trajectory = ada->getArm()->planToConfiguration(ada->getArm()->getNamedConfiguration("home_config"),ada->getArm()->getWorldCollisionConstraint());
+  bool success = true;
+  auto future = ada->getArm()->executeTrajectory(trajectory); // check velocity limits are set in FeedingDemo
+  try
   {
-    auto retval = moveAbove(
```

Comment:
Interesting that `MoveAbovePlate` no longer calls `moveAbove`, but `MoveAboveFood` does. Why the discrepancy? (Understanding the utility or lack thereof of `moveAbove` versus the other actions can help us as we think about refactoring the actions/subactions)

Reply:
Because `MoveAbovePlate` moves to a fixed configuration, and doesn't actually move above a fixed position.